packages/core/src/components/forms/_helper-text.scss

```diff
@@ -0,0 +1,118 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+@import "../../common/mixins";
+
+/*
+Helper text
```

Comment:
i think this is actually about "Form groups," which happen to support helper text.